frontend/src/components/navbar/NavLink.jsx

```diff
@@ -31,8 +31,9 @@ export default function NavLink({
       {dropdown !== undefined && dropdownOpen && (
         <div className="flex flex-col absolute flex-center bg-gray-200 w-[10rem] shadow-md z-[99]">
           {dropdown.map((nestedDropdown) => (
-            <span key={nestedDropdown.name} className="p-2 font-bold z-[99]">
+            <span className="p-2 font-bold z-[99]" key={nestedDropdown.name}>
               <NavLink
+                key={nestedDropdown.name}
```

Comment:
```suggestion
```
since the span is being mapped directly, the NavLink shouldn't need a key if i'm not mistaken